napari/layers/base/base.py

```diff
@@ -486,34 +488,45 @@ def position(self, position):
         self._update_value_and_status()
 
     @property
-    def dims(self):
-        warnings.warn(
-            (
-                "The layer.dims parameter is deprecated and will be removed in version 0.4.3."
-                " Instead you should use the viewer.dims parameter on the main viewer object."
-            ),
-            category=DeprecationWarning,
-            stacklevel=2,
-        )
-        return self._dims
+    def _dims_displayed(self):
+        """To be removed displayed dimensions."""
```

Comment:
Can you add a more elaborate note for "to be removed"? Maybe a link to discussion about this? "ultimately we aim to remove all slicing information from the layer itself, see (link)"?

Reply:
added